lib/puppet/parameter/value_collection.rb

```diff
@@ -181,11 +181,11 @@ def validate(value)
     return if empty?
 
     unless @values.detect { |name, v| v.match?(value) }
-      str = "Invalid value #{value.inspect}. "
+      str = _("Invalid value #{value.inspect}. ")
 
-      str += "Valid values are #{values.join(", ")}. " unless values.empty?
+      str += _("Valid values are #{values.join(", ")}. ") unless values.empty?
 
-      str += "Valid values match #{regexes.join(", ")}." unless regexes.empty?
+      str += _("Valid values match #{regexes.join(", ")}.") unless regexes.empty?
 
```

Comment:
You're totally right, these all got missed, probably because this PR is supposed to target the "node" directory, but these are in "parameter". No idea how that happened. Oh well, fixing.